addons/account_edi_ubl_cii_tax_extension/models/account_edi_common.py

```diff
@@ -84,7 +84,7 @@ def _get_tax_unece_codes(self, invoice, tax):
         if tax.ubl_cii_tax_category_code:
             reason_code = tax.ubl_cii_tax_exemption_reason_code
             reason_code = FIX_WRONG_CODES_MAPPING.get(reason_code, reason_code)
-            tax_exemption_reason = TAX_EXEMPTION_MAPPING.get(reason_code)
+            tax_exemption_reason = TAX_EXEMPTION_MAPPING.get(reason_code, _("Exempt from tax") if tax._requires_exemption_reason() else None)
```

Comment:
Does it mean we will end with `_("Exempt from tax")` after choosing `S - Standard Rate` if there is no `ubl_cii_tax_exemption_reason_code` ?